src/query/functions-v2/tests/it/scalars/math.rs

```diff
@@ -68,3 +73,38 @@ fn test_trigonometric(file: &mut impl Write) {
         Column::from_data(vec![1i64, -1, 1024]),
     )]);
 }
+
+fn test_exp(file: &mut impl Write) {
+    run_ast(file, "exp(2)", &[]);
+    run_ast(file, "exp(-2)", &[]);
+    run_ast(file, "exp(0)", &[]);
+}
+
+fn test_round(file: &mut impl Write) {
+    run_ast(file, "round(-1.23)", &[]);
+    run_ast(file, "round(1.298, 1)", &[]);
+    run_ast(file, "round(1.298, 0)", &[]);
+    run_ast(file, "round(23.298, -1)", &[]);
+    run_ast(file, "round(0.12345678901234567890123456789012345, 35)", &[
```

Comment:
Maybe need add some test like this:

run_ast(file, "atan2(a, 4)", &[(
        "a",
        DataType::Int64,
        Column::from_data(vec![1i64, -1, 1024]),
    )]);